src/abi.ts

```diff
@@ -75,3 +75,466 @@ export const TOKEN_ABI = [
     type: 'function',
   },
 ]
+
+export const L2_STANDARD_TOKEN_ABI = [
```

Comment:
You could likely grab this from `yarn add @eth-optimism/contracts-bedrock`

Reply:
Since bedrock is not in prod should we use `@eth-optimism/contracts`? I also tried using `@eth-optimism/contracts-bedrock`, but `getContractInterface('L2StandardERC20')` did not return anything

Reply:
Yea some stuff hasn't migrated over yet eventually the contracts team will rename contracts-bedrock contracts and contracts won't exist